src/contexts/applicationContext.tsx

```diff
@@ -65,10 +68,10 @@ function ApplicationProvider({ children }: ApplicationProviderProps) {
         error(e)
         toast({
           title: 'Error updating macro data',
-          description:
-            'Unable to update macro data, please re-open the app. If that does not work, please contact us on Discord.',
+          description: `Unable to update macro data: ${e}. 
+            Your system action filepath or website URL may be incorrect. Alternatively, please contact us on Discord.`,
           status: 'error',
-          duration: 2000,
+          duration: 10000,
```

Comment:
added a more descriptive error messages with a better duration so there is a chance to read it fully and or take a screenshot

Reply:
Error duration and closeability is being addressed lower down in the comment chain.